FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -80,6 +80,18 @@ - (BOOL)compareSnapshotOfView:(UIView *)view
                                        error:errorPtr];
 }
 
+- (BOOL)referenceImageRecordedInDirectory:(NSString *)referenceImagesDirectory
+                               identifier:(NSString *)identifier
+                                    error:(NSError **)errorPtr
+{
+    _snapshotController.referenceImagesDirectory = referenceImagesDirectory;
```

Comment:
Can we assert here like the other methods [do](https://github.com/Grubas7/ios-snapshot-test-case/blob/master/FBSnapshotTestCase/FBSnapshotTestCase.m#L40) ?